<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>975</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>639</width>
    <height>975</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>639</width>
    <height>975</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Green-Farm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>639</width>
      <height>114</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="headertext">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>24</y>
       <width>518</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Smart - Green Farm</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="footer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>864</y>
      <width>639</width>
      <height>114</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#280F23</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="re_btn">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>37</y>
       <width>128</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleDescription">
      <string>reBtn</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA;
background-color:#4A1942;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Refreash</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reportbtn">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>37</y>
       <width>128</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleDescription">
      <string>repBtn</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA;
background-color:#4A1942;
border-radius: 15px
</string>
     </property>
     <property name="text">
      <string>Report</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closebtn">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>40</y>
       <width>128</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleDescription">
      <string>cloBtn</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA;
background-color:#4A1942;
border-radius: 15px
</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>641</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#893168
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="tem1">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>62</y>
       <width>225</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>In Temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="tem2">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>168</y>
       <width>225</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>Out Temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="hum1">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>274</y>
       <width>225</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>In Humidity</string>
     </property>
    </widget>
    <widget class="QLabel" name="hum2">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>385</y>
       <width>225</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>Out Humidity</string>
     </property>
    </widget>
    <widget class="QLabel" name="uv1">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>507</y>
       <width>281</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>UV InConcentration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>612</y>
       <width>301</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#EAEAEA
</string>
     </property>
     <property name="text">
      <string>UV OutConcentration</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd1">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>70</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string>inTemp</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
text-align: center;</string>
     </property>
     <property name="text">
      <string>25</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd2">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>173</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string>outTemp</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
</string>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd3">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>273</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string>inHum</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
</string>
     </property>
     <property name="text">
      <string>3.6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd4">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>395</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string>outHem</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
</string>
     </property>
     <property name="text">
      <string>2.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd5">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>515</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleDescription">
      <string>UVin</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
</string>
     </property>
     <property name="text">
      <string>4.2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnd6">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>622</y>
       <width>83</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleDescription">
      <string>UVout</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #280F23;
opacity: 0.5;
border-radius: 15px;
color: #EAEAEA;
</string>
     </property>
     <property name="text">
      <string>3.4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>58</y>
       <width>62</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/iconfinder_weather-21_1530373.png);
border:0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>162</y>
       <width>62</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/iconfinder_weather-21_1530373.png);
border:0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>264</y>
       <width>62</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/iconfinder_weather_12_2682839.png);
border:0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>385</y>
       <width>62</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/iconfinder_weather_12_2682839.png);
border:0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>500</y>
       <width>68</width>
       <height>69</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/uv-index (5).png);
border:0
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>608</y>
       <width>68</width>
       <height>69</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/uv-index (5).png);
border:0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>21</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(11, 13, 112)</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>150</y>
       <width>21</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(11, 13, 112)</string>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
